mirrord/kube/src/api/kubernetes.rs

```diff
@@ -199,20 +199,13 @@ impl AgentManagment for KubernetesAPI {
     where
         P: Progress + Send + Sync,
     {
-        let runtime_data = if let Some(ref path) = self.target.path {
+        let runtime_data = if let Some(ref path) = self.target.path && !matches!(path, mirrord_config::target::Target::Targetless) {
             let runtime_data = path
                 .runtime_data(&self.client, self.target.namespace.as_deref())
                 .await?;
 
             Some(runtime_data)
         } else {
-            // Most users won't see this, since the default log level is error, and also progress
```

Comment:
Why'd you remove this? Also is running the CLI without specifying a target still supported?

Reply:
Because as the comment says, it's never really seen. If we want users to know that, we need to change it to use the progress API.
Yes, it's still supported.